firestore/integration_test_internal/src/bundle_test.cc

```diff
@@ -0,0 +1,196 @@
+#include <string>
```

Comment:
Please add tests for `NamedQuery()`, especially specifying an empty and a non-existent name.